site/README.md

```diff
@@ -0,0 +1,59 @@
+# Website for [Kubeapps](https://kubeapps.com/)
+
+## Deployment
+
+The website will be deployed to production each time a new commit gets merged into the main branch. It is deployed using Netlify. The [netlify.toml](./netlify.toml) file holds the configuration.
+
+## Requirements
+
+This site uses [Hugo](https://github.com/gohugoio/hugo) for rendering. It is recommended you run `hugo` locally to validate your changes render properly.
+
+### Local Hugo Rendering
+
+Hugo is available for many platforms. It can be installed using:
+
+- Linux: Most native package managers
+- macOS: `brew install hugo`
+- Windows: `choco install hugo-extended -confirm`
+
+Once installed, you may run the following from the `/site` directory to access a rendered view of the documentation:
+
+```bash
+cd site
+hugo server --disableFastRender
+```
+
+Access the site at [http://localhost:1313](http://localhost:1313). Press `Ctrl-C` when done viewing.
+
+The [site/content/docs/latest](./content/docs/latest) directory holds the project documentation whereas [site/themes/template/static../img/docs](./themes/template/static../img/docs) contains the images used in the documentation. Note they have to be under that folder to be properly served.
+
+## Check writing
+
+In order to validate and ensure a proper style of writing, it is recommended to run the [vale validator](https://vale.sh/docs/vale-cli/installation/) with a set of [style rules](https://github.com/errata-ai/styles). The rules are present in the project codebase in the directory. Some of them have been slightly modified to fit our project needs.
```

Comment:
The rules have been added to the repo, under the folder `styles`

Reply:
Excellent. Assuming people can add exceptions intentionally if needed, it'd be great to have the build fail if the validator picks a breach of the style rules?

Reply:
Well... there are plenty of pre-defined warnings and errors. I have disabled some of them, but others remain as is. If we happen to add it to the building process, certainly we can tweak the rules and determine which are errors vs warnings for us. 
Unfortunately, AFAIK, there is no way to set sth as an exception, but we can convert it into a warning.

Here's a example of the output:

```

 content\docs\latest\tutorials\using-an-OIDC-provider.md
 1:1   warning  Try to keep the Flesch reading  Readability.FleschReadingEase
                ease score (39.42) above 40
                (college).
 3:28  error    Using 'simple' may come across  alex.Condescending
                as condescending.
 14:1  error    Don't start a sentence with     write-good.ThereIs
                'There are'.
 48:1  error    Don't start a sentence with     write-good.ThereIs
                'There are'.


 content\docs\latest\tutorials\managing-carvel-packages.md
 89:128  error  Don't start a sentence with     write-good.ThereIs
                'There are'.
 215:45  error  Using 'easily' may come across  alex.Condescending
                as condescending.
```

For instance, with this report I would try to reduce the complexity of the sentences in `using-an-OIDC-provider.md` and rephrase some sentences to remove, if possible, some  `easily` and `simple` words. However, I don't see it is really a problem to start with `there are....`. Perhaps I'd remove that rule or, at least, mark it as a warning, not an error